[IMP] Documents: email alias (#10024)
odoo/documentation: content/applications/productivity/documents.rst

You can share a link to a workspace and allow users to :guilabel:`Download` its content or
:guilabel:`Download and Upload` files to it.

> To do so, select the workspace to share in the left column of your dashboard and possibly one or
> several tags that will be automatically added to the uploaded documents. Then, click the arrow next
> to the :guilabel:`Upload` button and :guilabel:`Share`.

Reviewer: ```suggestion
To do so, go to the left column of your dashboard and select the workspace you want to share. You can also choose one or more tags that will be automatically added to the uploaded documents.. Then, click the arrow next
to the :guilabel:`Upload` button and :guilabel:`Share`.
```
IMO "select" is a bit too far from the tags part, making the sentence unclear. To avoid having two "and"s close together (which would be the result of reverting to "Go to the left column and select a workspace and possibly one or several tags"), I think breaking it into two separate sentences like I did in my suggestion works better.

Reviewer: @Donapi I noticed that this paragraph seems to be missing after the last force-push (I think it should be after line 235?). Could it have been accidentally omitted during the rebase or while resolving conflicts?

.. _documents/upload:

Files upload
^^^^^^^^^^^^

Toggle the :guilabel:`Allow upload` switch to allow users to upload files to your workspace.

Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
the workspace.

.. note::

   Your **email server** must be configured to be able to create an **email alias**. If it isn't
   configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the

>    :doc:`mail server <../../applications/general/email_communication/email_servers>`.

Reviewer: I think you can remove the blank line one line 254 as it's not necessary.

And sorry for insisting, but I really think the information gets lost between the two note boxes (+ the tip below the second one).
![image](https://github.com/user-attachments/assets/e7fd7e42-0226-482c-930d-410128f34699)

I know you mentioned that people might be lost if they haven't configured the email server yet, but I believe it should be fine. There's a good chance they will have already configured the server before reaching this feature anyway. Could we move the info to the second note box, as another list item, please?

.. _documents/upload:

Files upload
^^^^^^^^^^^^

Toggle the :guilabel:`Allow upload` switch to allow users to upload files to your workspace.

Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
the workspace.

Enter an alias in the :guilabel:`Email Alias` field. All documents sent to this email address are
uploaded to the workspace using the chosen tags.

.. note::
   - Your **email server** must be configured to be able to create an **email alias**. If it isn't
     configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the
     :doc:`mail server <../../applications/general/email_communication/email_servers>`;
   - By default, the :guilabel:`Document Owner` is the person who uploads a file to a workspace, but
     you can select another user. You can also set a :guilabel:`Contact`, usually an external

>      person, such as a partner;

Reviewer: and these can get a period at the end since they're full sentences :slightly_smiling_face:

.. _documents/upload:

Files upload
^^^^^^^^^^^^

> Switch on the :guilabel:`Allow upload` button to allow users to upload files from your workspace.

Reviewer: ```suggestion
Toggle the :guilabel:`Allow upload` switch to allow users to upload files to your workspace.
```

Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
the workspace.

.. note::

   Your **email server** must be configured to be able to create an **email alias**. If it isn't
   configured yet:

   #. Click :guilabel:`Choose or Configure Email Servers`;
   #. In the :guilabel:`General Settings` page, scroll down to the :guilabel:`Discuss` section and
      go to the :guilabel:`Alias Domain`;
   #. Enter your email server details and :guilabel:`save`.

Enter an alias in the :guilabel:`Email Alias` field. All documents sent to this email address are

> uploaded to the workspace using the chosen tags.

Reviewer: I think this is actually described (and to a greater extent since this also depends on the hosting type here), I think it's be better to just link to the email servers doc, and maybe even this part of the doc: https://www.odoo.com/documentation/master/applications/general/email_communication/email_servers.html#manage-inbound-messages

   Your **email server** must be configured to be able to create an **email alias**. If it isn't
   configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the mail server/your alias domain [ref or doc --> see my comment below] .

I'm not a 100% sure here since I don't know this topic very well but  could we say "Your **alias domain** must be configured xxx" ??
- If not: I suggest we keep use: :doc:`configure the mail server <link to the mail server doc>`
- If yes: I suggest we use: :ref:`configure your alias domain <ref to the part of the mail server doc where we explain email aliases>`

Author: FRGI prefers to use mail server, updated as per your suggestion.
